frappe/public/js/frappe/views/communication.js

```diff
@@ -354,11 +354,14 @@ frappe.views.CommunicationComposer = Class.extend({
 		var fields = this.dialog.fields_dict;
 		var attach = $(fields.select_attachments.wrapper).find(".attach-list").empty();
 
-		if (cur_frm){
+		if (this.attachments.length) {
```

Comment:
you have not manually tested this... this will break if `this.attachments` is null.

@ci2014 We appreciate you sending contributions, but PLEASE manually test before sending pulls because you will break things for everyone else and you are adding to everyone else workload.

You are directly editing in github which is also a bad idea for such stuff.

Over time if you want to be a contributor, you have to build a reputation, so be careful of what pulls you are sending.

Reply:
@rmehta Thank you for your note. You are right, if it's null, it will raise an error. However it is manually tested, like all of my PRs, since they are features, which I need in development progress. Of course, I will test it more in future.